api/src/main/java/org/openmrs/module/aijar/AijarActivator.java

```diff
@@ -316,6 +319,10 @@ private void generateOpenMRSIdentifierForPatientsWithout() {
         // mapping for creating visits without encounters to the default facility visit type
         properties.add(new GlobalProperty("emrapi.EmrApiVisitAssignmentHandler.encounterTypeToNewVisitTypeMap", "default:7b0f5697-27e3-40c4-8bae-f4049abfb4ed"));
 
+
+        // adding the master health facility number for health center place holder
```

Comment:
It needs to be refactored but it currently works - so no need to change it now when we are under pressure